geometry_msgs/PoseStamped[] trajectory
float64 duration
bool enable
